app/emoji-custom/server/methods/listEmojiCustom.js

```diff
@@ -2,7 +2,16 @@ import { Meteor } from 'meteor/meteor';
 import { EmojiCustom } from '/app/models';
 
 Meteor.methods({
-	listEmojiCustom(options = {}) {
-		return EmojiCustom.find(options).fetch();
+	listEmojiCustom(options = {}, updatedAt) {
```

Comment:
since the method response has changed, the [client code that uses it](https://github.com/RocketChat/Rocket.Chat/blob/develop/app/emoji-custom/client/lib/emojiCustom.js#L162) needs to be changed as well